fastkafka/_cli.py

```diff
@@ -33,15 +33,22 @@ def run(
         ...,
         help="input in the form of 'path:app', where **path** is the path to a python file and **app** is an object of type **FastKafka**.",
     ),
+    kafka_broker: Optional[str] = typer.Option(
+        None, help="broker to use from kafka_brokers"
```

Comment:
kafka_broker, one of the keys of the kafka_brokers dictionary passed in the constructor of FastaKafka class.

Reply:
Done